pkgs/by-name/ro/rofi-unwrapped/package.nix

```diff
@@ -91,22 +94,29 @@ stdenv.mkDerivation rec {
     xcbutilxrm
   ];
 
-  mesonFlags =
-    lib.optionals x11Support [ "-Dimdkit=true" ]
-    ++ lib.optionals (!waylandSupport) [ "-Dwayland=disabled" ]
-    ++ lib.optionals (!x11Support) [ "-Dxcb=disabled" ];
+  mesonFlags = [
+    (lib.mesonBool "imdkit" x11Support)
+    (lib.mesonEnable "wayland" waylandSupport)
+    (lib.mesonEnable "xcb" x11Support)
+  ];
 
   doCheck = false;
 
-  meta = with lib; {
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = "--version";
```

Comment:
It looks like you also need to set `doInstallCheck = true;` for this to run (they are skipped by default)

ref: https://nixos.org/manual/nixpkgs/stable/#variables-controlling-the-installcheck-phase

Reply:
True, and I know it! Completely forgot this time... good catch!